style.css

```diff
@@ -956,6 +956,50 @@ ul {
   padding: 10px 0 0;
 }
 
+.hero .search {
+  overflow: hidden;
+}
+
+.hero .search:before {
+  right: 125px;
+}
+
+@media (min-width: 768px) {
```

Comment:
If this is only scoped to `.hero .search:before` we can move this into the block above.